tools/sst_scan.cc

```diff
@@ -241,13 +267,36 @@ main(
                                     if (riak_translations && '\x10'==*parsed.user_key.data())
                                     {
                                         leveldb::Slice cursor_slice;
+                                        std::string type, bucket;
```

Comment:
These variables are not currently used, due to the "#if 0" blocks below; should probably comment out this line for now.